doc/releasing.md

```diff
@@ -9,6 +9,7 @@ Version numbers of libmongocrypt must follow the format 1.[0-9].[0-9] for releas
 Do the following when releasing:
 - Ensure `etc/purls.txt` is up-to-date. 
 - If this is a feature release (e.g. `x.y.0` or `x.0.0`), follow these steps: [Creating SSDLC static analysis reports](https://docs.google.com/document/d/1rkFL8ymbkc0k8Apky9w5pTPbvKRm68wj17mPJt2_0yo/edit).
+- Ensure necessary issues are addressed in the `mongodb/libmongocrypt` target in [Snyk](https://app.snyk.io/). See [Checking Snyk](https://docs.google.com/document/d/1SEUAuF923dVPL_Oq4P-TNXEby8u8fVOCcfwYrtHA-YQ/edit).
```

Comment:
May want to reword/defer this instruction to reference the augmented SBOM obtained from Silk instead, given Silk will handle communicating Snyk-detected vulnerabilities via the augmented SBOM.

Reply:
Snyk _may_ require manually curating the list of detected projects whose 3PDs are being reported (deactivation or deletion). This step may be worth documenting instead. (e.g. see proposed wording for [CXX Driver release instructions](https://github.com/mongodb/mongo-cxx-driver/blob/dea9db443a3dae7155f03d0a966d8dc261f11853/etc/releasing.md#check-and-update-snyk)).

Reply:
> manually curating the list of detected projects whose 3PDs are being reported (deactivation or deletion).

Some libmongocrypt projects not relevant to the C library appear relevant to language bindings. Rather than recommending deactivating or deleting, these steps refer to the CLI target created for tracking only the C library.